.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "singleQuote": false,
```

Comment:
I had to add this file or I ended up with conflicting prettier rules which changed everything on save